docs/excel2xml-module.md

```diff
@@ -6,10 +6,10 @@
 
 There are two kinds of Excel files that can be transformed into an XML file:
 
-| structure        | provenance  | tool                     | example screenshot                                       |
-|------------------|-------------|--------------------------|----------------------------------------------------------|
-| custom structure | customer    | module `excel2xml`       | ![](./assets/images/img-excel2xml-raw-data-category.png) |
-| DSP structure    | DSP server  | CLI command `excel2xml`  | ![](./assets/images/img-excel2xml-closeup.png)           |
+| structure        | provenance | tool                    | example screenshot                                       |
+| ---------------- | ---------- | ----------------------- | -------------------------------------------------------- |
+| custom structure | customer   | module `excel2xml`      | ![](./assets/images/img-excel2xml-raw-data-category.png) |
+| DSP structure    | DSP server | CLI command `excel2xml` | ![](./assets/images/img-excel2xml-closeup.png)           |
```

Comment:
Sorry, autoformatting plugin that changed its behavior and reformatted all tables...